core/protos/o2control.proto

```diff
@@ -233,6 +233,9 @@ message ControlEnvironmentReply {
     string id = 1;
     string state = 2;
     uint32 currentRunNumber = 3;
+    int64 StartOfTransition = 4;
+    int64 EndOfTransition = 5;
+    int64 TransitionDuration = 6;
```

Comment:
@claireguyot could you please make these lowercase for consistency? You should then regenerate the code, though I expect the output to be identical.